rust/bridge/shared/src/ffi/storage.rs

```diff
@@ -77,7 +77,10 @@ impl IdentityKeyStore for &FfiIdentityKeyStoreStruct {
         Ok(IdentityKeyPair::new(IdentityKey::new(pub_key), *priv_key))
     }
 
-    async fn get_local_registration_id(&self, ctx: Context) -> Result<u32, SignalProtocolError> {
+    async fn get_local_registration_id(
+        &self,
+        ctx: Context,
+    ) -> Result<SessionSeed, SignalProtocolError> {
```

Comment:
I'm not sure about the name "SessionSeed". That's how it's used, sure, but it already has a name: "RegistrationId".

Reply:
Fixed!